packages/@react-spectrum/list/package.json

```diff
@@ -52,6 +52,7 @@
     "@react-spectrum/text": "^3.1.8",
     "@react-spectrum/textfield": "^3.4.0",
     "@react-spectrum/utils": "^3.6.8",
+    "@react-spectrum/view": "^3.1.8",
```

Comment:
extra?

Reply:
I think this just didn't make it in the merge commit, but should be there. We have a story with `<Content>No results found</Content>` so it looks right

Reply:
o, for stories we don't need this, unless the actual code for lists depends on it, it should not be here
if something in storybook is complaining, we can put it into devDeps

Reply:
Ahh thats right, just removed